tests/test_app.py

```diff
@@ -293,6 +306,7 @@ class Win(mock.MagicMock):
         assert ex.call_count == 1
         assert mock_event_loop.call_count == 1
         assert mock_worker.call_count == 1
+        assert mock_set_except.call_count == 1
```

Comment:
when the app runs the exception hook setup call should happen once and once only.